main/lsp/LSPTypechecker.cc

```diff
@@ -247,7 +247,7 @@ vector<core::FileRef> LSPTypechecker::runFastPath(LSPFileUpdates &updates, Worke
     fast_sort(subset);
     subset.resize(std::distance(subset.begin(), std::unique(subset.begin(), subset.end())));
 
-    config->logger->debug("Taking fast path");
+    config->logger->debug("Running fast path");
```

Comment:
Was this change intentional? (I have no objection to it if so.)

Reply:
yeah, there's another log line that's identical, and i wanted this to be different.